docs/content/en/docs/clustermgmt/support/cluster-license.md

```diff
@@ -2,16 +2,79 @@
 
-When you purchase an EKS Anywhere Enterprise Subscription, licenses for your clusters are provisioned in [AWS License Manager](https://docs.aws.amazon.com/license-manager/latest/userguide/license-manager.html) in the AWS account you used to purchase the subscription. After purchasing your subscription, navigate to the AWS License Manager console and accept the license grants following the steps in the [AWS License Manager documentation](https://docs.aws.amazon.com/license-manager/latest/userguide/granted-licenses.html). Save the License ID strings for your licenses, as you will need them to license your clusters.
+### View licenses for an EKS Anywhere subscription
+
+You can view the licenses associated with an EKS Anywhere Enterprise Subscription in the [Amazon EKS Console.](https://console.aws.amazon.com/eks/home#/eks-anywhere)
+
+Follow the step below to view EKS Anywhere licenses with the AWS CLI.
+
+**Get license ARNs based on subscription `name` with the AWS CLI**
+- Replace `region-code` with the AWS Region that hosts your subscription (for example `us-west-2`).
+- Replace `my-subscription` in the `--query` string with the `name` for your subscription.
+
+```bash
+aws eks list-eks-anywhere-subscriptions \
+  --region 'region-code' \
+  --query 'subscriptions[?name==`my-subscription`].licenseArns[]'
+```
+
+The License ID is the last part of the ARN string. For example, the License ID is shown in bold in the following example: *arn:aws:license-manager::12345678910:**license:l-4f36acf12e6d491484812927b327c066***
+
+**Get license details based a license ARN with the AWS CLI**
+
+- Replace `region-code` with the AWS Region that hosts your subscription (for example `us-west-2`).
+- Replace `my-license-arn` with the license ARN returned from the previous command.
+
+```bash
+aws license-manager get-license \
+  --region 'region-code'
+  --license-arn 'my-license-arn'
+```
+
+<details>
+  <summary>Expand for sample command output</summary>
+  <br /> 
+  {{% content "get-license-output.md" %}}
+</details>
+
+### Accept EKS Anywhere license grant
+
+You can accept the license grants associated with an EKS Anywhere Enterprise Subscription in the [AWS License Manager Console](https://console.aws.amazon.com/license-manager/) following the instructions in the [AWS License Manager documentation](https://docs.aws.amazon.com/license-manager/latest/userguide/granted-licenses.html). 
+
+See the steps below for accepting EKS Anywhere license grants with the AWS CLI.
+
+**Get grant ARNs based on subscription `name` with the AWS CLI**
+
+- Replace `region-code` with the AWS Region that hosts your subscription (for example `us-west-2`).
+- Replace `my-subscription` in the `--query` string with the `name` for your subscription.
+
+```bash
+aws license-manager list-received-licenses \
```

Comment:
we need to provide the list of license arns to this. the page size of this resposne is only 20 or something. if a customer has a lot of licenses, might not filter correctly.

```
aws license-manager list-received-licenses \
  --license-arns "arn1" "arn2"
  --region 'region-code' \
  --query 'Licenses[?LicenseName==`EKS Anywhere license for subscription my-subscription`].LicenseMetadata[].Value'

Reply:
I added a `--filter 'Name=IssuerName,Values=Amazon EKS Anywhere'` ... that should be sufficient for most use cases